cmd/tempo-serverless/go.mod

```diff
@@ -33,3 +33,6 @@ replace (
 // This function is used by jeagertracing/jaeger, but we can't update jaeger
 // without updating the open-telemetry/collector as well
 replace google.golang.org/grpc => google.golang.org/grpc v1.38.0
+
+// these have been added due to cortex incompatibilies. they can be removed once we drop cortex as a dependency
```

Comment:
i thought this PR dropped all remaining cortex deps? or are there some still out there?

Reply:
Well, yes good catch. :)  There are two references currently.  Here I was thinking that we change the dependencies after the removal, but if you would prefer to update here, I can do that also.

Two of these just look like version pinings, but the other two look like forks we might want to trawl through the code for also.

Reply:
This PR is already insane. I agree. Let's get this in and then we can do a separate cleanup on the go.mod file.